0009-non-helium-hotspots.md

```diff
@@ -0,0 +1,248 @@
+
+
+                                                        +------------+
+                                                        |            |
+                                                +------>+  Level 3A  +---------+
+                                                |       |            |         |
+    +-----------+         +-----------+         |       +------------+         |         +-----------+
+    |           |         |           |         |                              |         |           |
+    |  Level 1  +-------->+  Level 2  +-------->+                              +-------->+  Level 4  |
+    |           |         |           |         |                              |         |           |
+    +-----+-----+         +-----------+         |       +------------+         |         +-----------+
+          |                                     |       |            |         |
+          |                                     +------>+  Level 3B  +---------+
+          |                                             |            |
+          |                                             +------+-----+
+          |                                                    ^
+          |                                                    |
+          |                                                    |
+          +----------------------------------------------------+
+
+
+Before we get into the details and constraints of each level, we need to set some common ground:
+
+* Every hotspot joining the network starts at the first level.
+* Every successive level is granted the priviledges of all previous levels.
+* Organic network growth allows hotspots to get to level 3A. In order to reach level 4, the owner must stake HNT.
+* There is a path to get to level 3B from level 1 via on-chain governance mechanism (TBD).
+
+### Level I: Getting on the network
+
+- Every hotspot which joins the helium network starts at Level 1 by issuing an `add_gateway` transaction.
+- The entry fee is set to burning data credits worth $10 USD.
+- A hotspot at Level 1 does not have any location asserted on the network, however, it is allowed to witness nearby PoC
+  challenges. This is to ensure that when the hotspot does assert a location via `assert_location` transaction, we can
+  be confident about its claim of location by analyzing its witness list.
+- A hotspot at Level 1 is allowed to transmit data and earn HNT based only on transmitting network data.
+
+### Level 2: Asserting location on the network
+
+- A hotspot must satisfy the below minimum criteria to enter Level 2:
+    * It must be a Level 1 hotspot for X number of blocks.
+    * Issue an `assert_location` transaction by incurring a $40 USD data credit burn fee.
+    * Witness sufficient number of PoC challenges involving Level 3 and above hotspots.
+- A hotspot at Level 2 gains access to participate in PoC challenges (become a member in the PoC path).
```

Comment:
can we include demotion qualifiers here as well? "An L2 Hotspot will be demoted to L1 if it fails n challenges" (in a row, over n blocks, whatever)

Reply:
what happens to an L2's `assert_location` txn if it gets demoted?

Reply:
I defined a cursory level demotion criteria in a section below